test/deps/binaryen/binaryen_test.cpp

```diff
@@ -84,3 +86,48 @@ TEST(BinaryenTest, Example1) {
   // dispose module
   delete wasm;
 }
+
+/**
+ * @given WebAssembly S-expression code exporting a function
+ * (sumtwo) taking two arguments of type i32 returning their sum of type i32
+ * @when this code is interpreted using Binaryen
+ * @then sumtwo implementation on wasm is invoked from C++ with given arguments
```

Comment:
That line probably should go at `@when` section. Here should be written that the received result is really a requested sum

Reply:
Ok